[WT-225] bugfix/Add files and folders downlodad using blob in firefox

CandelR opened the pull request:
- Fixed 500 error when call `'/storage/tree/:folderId'` endpoint correcting the way that `include` param was set.  
   - In the function `getFilesByFolderId`, the `include` param was added inside of `where` param in `Model.file.findAll` function, but it has to go at same level. This fixes a crash that happens when`getFilesByFolderId` is executed when calling the above-mentioned enpoint because the wrong addition of `include` param.
   - Code lines affected → 230 to 242
   - Documentation: 
      - [Sequelize findAll ](https://sequelize.org/api/v6/class/src/model.js~model#static-method-findAll)

Warning! ⚠️ 
 - It is necessary to merge this PR before [the related one](https://github.com/internxt/drive-web/pull/510)
 
 Related PRs
  - https://github.com/internxt/drive-web/pull/510

sg-gs commented:
Hello @CandelR, as far as I can see, these changes are only lint fixes. If something has changed, please refer it somewhat to me as there is just one commit.

Just take into consideration for the future to split lint fixes from other changes in different commits in order to make it easier, cleaner and faster to review what are really the changes introduced.

Also, please explain and add to the description of this PR why this: 

_- Fixed the incorrect way the includes attribute was set._

 Fixes something, what fixes, and what are the consequences, so in the future, we can revisit this if needed without having to compare each line to see what was not working and why this fix was needed.

fabioespinosa commented:
We should do one full prettier —write . So we avoid reviewing them in PRs 

we would lose the git lens feature to see who made what though 🤔

CandelR commented:
> Hello @CandelR, as far as I can see, these changes are only lint fixes. If something has changed, please refer it somewhat to me as there is just one commit.
> 
> Just take into consideration for the future to **split lint fixes from other changes in different commits** in order to make it easier, cleaner and faster to review what are really the changes introduced.
> 
> Also, please explain and add to the description of this PR why this:
> 
> _- Fixed the incorrect way the includes attribute was set._
> 
> Fixes something, what fixes, and what are the consequences, so in the future, we can revisit this if needed without having to compare each line to see what was not working and why this fix was needed.

@sg-gs sorry about that! I have changed the description of the PR giving more detailed information on what it fixes and why it was not working and I have removed the Lint changes that generated noise in the PR. 
I will take into consideration for future PRs to separate the Lint changes with the task related ones! Thank you!

sg-gs commented:
There is no issue with adding lint fixes guys @fabioespinosa @CandelR, just splitting them into different commits makes it easier to differentiate
